LitvischenkoYevhen/HW5/HW5_task_2.py

```diff
@@ -0,0 +1,8 @@
+# Print Fibnacci numbers
+num = int(input('Enter the number: '))
```

Comment:
you should use meaningful variable names

Reply:
Thank you/ I will try )